app/components/chat/tv-mode-button.hbs

```diff
@@ -1,8 +1,18 @@
-{{#if this.videoStream.active}}
-  <button
-    class="cool-button"
-    title={{if this.showingViz (t "chat.titles.video_on") (t "chat.titles.video_off") }} type="button" {{on "click" this.toggleDisplay}}
+<div
+  class="grid grid-cols-2"
+  title={{if @enabled (t "chat.titles.video_on") (t "chat.titles.video_off") }}
+>
+  <label
+    id="tv-mode-input-label"
+    for="tv-mode-input"
   >
-    {{this.tvModeIcon}}
-  </button>
-{{/if}}
+    Show Video
+  </label>
+  <input 
+    type="checkbox"
+    name="tv-mode-input"
+    id="tv-mode-input"
+    {{on "change" @toggleVideo}}
+    checked=@enabled
```

Comment:
![2024-01-21-211929_885x349_scrot](https://github.com/datafruits/datafruits/assets/66243/c6d69582-dd03-4b32-8824-19b46bd856db)

When I have both settings windows open , looks like the checkbox on one doesn't update....even tho we are using the service so it seems like it should , weird.

Reply:
My suggestion below might fix it

Reply:
Thanks those fixed it. Hard to remember all these ember things 😅